packages/odyssey-react/src/components/Box/Box.module.scss

```diff
@@ -18,6 +18,7 @@
 @import "box-margin";
 @import "box-padding";
 @import "box-overflow";
+@import "box-text";
 @import "box-border";
 @import "box-background";
 @import "box-cursor";
```

Comment:
I like what youve done here, but if we intend to get rid of SCSS -- how will this work?

Reply:
@arnoldsandoval-okta for now we won't worry about this - we're still a ways off from removing SCSS ourselves but we've already effectively removed the requirement for downstream teams to **also** use SCSS which was more problematic in my mind.